examples/js/package.json

```diff
@@ -4,7 +4,7 @@
   "dependencies": {
     "react": "^18.1.0",
     "react-dom": "^18.1.0",
-    "spectacle": "*"
+    "spectacle": "workspace:*"
```

Comment:
What was the change that required adding this back? cc @ryan-roemer do you recall what we did to drop this?

Reply:
Let me pull the branch and take a stab at reverting this.

Reply:
Oh, sorry, I thought this was the correct way to do it. I was getting an error like `Field 'browser' doesn't contain a valid alias configuration`, and having seen this pattern elsewhere thought it might be the way to fix it, and it did indeed seem to work.
I don't remember exactly what made the error start to show again, sorry.

Reply:
So with:

```diff
diff --git a/examples/js/package.json b/examples/js/package.json
index 9d948b2..5c7ab39 100644
--- a/examples/js/package.json
+++ b/examples/js/package.json
@@ -4,7 +4,7 @@
   "dependencies": {
     "react": "^18.1.0",
     "react-dom": "^18.1.0",
-    "spectacle": "workspace:*"
+    "spectacle": "*"
   },
   "devDependencies": {},
   "scripts": {
diff --git a/examples/md/package.json b/examples/md/package.json
index 851de0b..cdce404 100644
--- a/examples/md/package.json
+++ b/examples/md/package.json
@@ -2,7 +2,7 @@
   "name": "spectacle-example-md",
   "private": true,
   "dependencies": {
-    "spectacle": "workspace:*",
+    "spectacle": "*",
     "react": "^18.1.0",
     "react-dom": "^18.1.0"
   },
diff --git a/examples/typescript/package.json b/examples/typescript/package.json
index 944a064..4ceb93a 100644
--- a/examples/typescript/package.json
+++ b/examples/typescript/package.json
@@ -2,7 +2,7 @@
   "name": "spectacle-example-ts",
   "private": true,
   "dependencies": {
-    "spectacle": "workspace:*",
+    "spectacle": "*",
     "react": "^18.1.0",
     "react-dom": "^18.1.0"
   },
diff --git a/packages/create-spectacle/package.json b/packages/create-spectacle/package.json
index 5e94b95..cb854a4 100644
--- a/packages/create-spectacle/package.json
+++ b/packages/create-spectacle/package.json
@@ -25,7 +25,7 @@
   "devDependencies": {
     "@types/node": "^18.0.3",
     "@types/prompts": "^2.0.14",
-    "spectacle": "workspace:*"
+    "spectacle": "*"
   },
   "resolutions": {},
   "scripts": {
```

Looking at the lock file we're still correctly linking:

```yml
  examples/js:
    specifiers:
      react: ^18.1.0
      react-dom: ^18.1.0
      spectacle: '*'
    dependencies:
      react: 18.2.0
      react-dom: 18.2.0_react@18.2.0
      spectacle: link:../../packages/spectacle

  examples/md:
    specifiers:
      react: ^18.1.0
      react-dom: ^18.1.0
      spectacle: '*'
    dependencies:
      react: 18.2.0
      react-dom: 18.2.0_react@18.2.0
      spectacle: link:../../packages/spectacle

# ...

  examples/typescript:
    specifiers:
      '@types/react': ^18.0.12
      '@types/react-dom': ^18.0.5
      react: ^18.1.0
      react-dom: ^18.1.0
      spectacle: '*'
    dependencies:
      react: 18.2.0
      react-dom: 18.2.0_react@18.2.0
      spectacle: link:../../packages/spectacle
    devDependencies:
      '@types/react': 18.0.17
      '@types/react-dom': 18.0.6

  packages/create-spectacle:
    specifiers:
      '@types/node': ^18.0.3
      '@types/prompts': ^2.0.14
      '@types/yargs': ^17.0.11
      chalk: ^4.1.2
      clear: ^0.1.0
      cli-spinners: ^2.6.1
      log-update: 4.0.0
      prompts: ^2.4.2
      spectacle: '*'
      yargs: ^17.5.1
    dependencies:
      '@types/yargs': 17.0.11
      chalk: 4.1.2
      clear: 0.1.0
      cli-spinners: 2.6.1
      log-update: 4.0.0
      prompts: 2.4.2
      yargs: 17.5.1
    devDependencies:
      '@types/node': 18.0.3
      '@types/prompts': 2.0.14
      spectacle: link:../spectacle
```

I think we can safely revert to `"spectacle": "*"` (and just double check we get `spectacle: link:../spectacle` in the lock file).